mail-mta/opensmtpd/opensmtpd-6.8.0_p2-r3.ebuild

```diff
@@ -1,9 +1,9 @@
 # Copyright 1999-2023 Gentoo Authors
```

Comment:
Could we do this one without git mv, as it's a lot of changes? Keep everything else as is, just un-git-mv (so new revision) and ekeyword ~all the new one.

Reply:
I changed it.